.eslintignore

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: 2023 TomTom <http://tomtom.com>
```

Comment:
Is it expected to have `http` instead of `https://www.tomtom.com/`?

Reply:
I did not dare to change it - however, I’d argue that n URL is not really a contact address and Id expect an email address to TomToms Open Source (or legal) team?